crates/revive-strategy/src/cheatcodes/mod.rs

```diff
@@ -655,6 +655,10 @@ fn select_revive(ctx: &mut PvmCheatcodeInspectorStrategyContext, data: Ecx<'_, '
 
     execute_with_externalities(|externalities| {
         externalities.execute_with(|| {
+            // Enable debug mode to bypass size checks during migration
+            let debug_settings = DebugSettings::new(true);
```

Comment:
should it be bypassed only for test contract? How to check now if the contract sizes are in the limit and will deploy to the real chain?